src/rgw/rgw_sal_motr.cc

```diff
@@ -67,6 +68,89 @@ static unsigned roundup(unsigned x, unsigned by)
   return ((x - 1) / by + 1) * by;
 }
 
+std::string base62_encode(uint64_t value, size_t pad)
```

Comment:
As i understand, this is the code coming from legacy s3server versioning feature implementation by @keithpine, please add link to that PR here in review comment

Reply:
Legacy s3 PR for sortable and globally unique version-ids - https://github.com/Seagate/cortx-s3server/pull/1686